package RuntimeExeptions;

public class App {
    public static void main(String[] args) {
        
        // String empty = null;
        // System.out.println(empty.length());

        String[] numbers = {"one", "two", "three"};
        System.out.println(numbers[3]);

        // jadi runtime exception adalah error yang disebabkan oleh kesalahan code
    }
}
